app/components/emailValidation/CorrespondingAuthorConfirmation.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Flex, Box } from 'grid-styled'
+import { Heading } from '@pubsweet/ui'
+import LoginButton from '@pubsweet-pending/login-orcid/Login'
+
+const CorrespondingAuthorConfirmation = ({ location }) => (
+  <Flex>
+    <Box>
+      <Heading level={1}>Thank you!</Heading>
+      <p>
+        You have been listed as the corresponding author for [title]. To track
+        the progress of this submission, please log in below.
+      </p>
+      <LoginButton location={location} />
```

Comment:
At the moment I'm using the same button that we take from a pubsweet package for our login page.

Unsure what "pubsweet-pending" is, or where it lives. Couldn't find any mention in docs anywhere (perhaps unsurprising, given the name :smile:). Hopefully we can just style it to look like the designs.
![image](https://user-images.githubusercontent.com/15656538/42229760-22dbf460-7edf-11e8-872b-f172e9997019.png)

Otherwise, may need to change this component.

Reply:
`@pubsweet-pending` is the made up scope name Peter suggested we use for packages which will most likely be contributed to PubSweet but are still in our monorepo.

Reply:
What are the product requirements for this login button? Does it do an Orcid login then associate the "virtual" user fro the email gateway with the "real" user from Orcid?

Reply:
@NickDuf What are your thoughts on this? It struck me that maybe we need to make clear on this button that it's a 'Log In with ORCiD'.

Product requirements are exactly as Tam suggests:

- User clicks the log in button and is redirected to ORCiD where they can log in with existing credentials or create a new account.

- Upon logging in the user is redirected to their dashboard, where the manuscript will now be listed along with an indication that they are the corresponding author.

- We imagine they will have access to a read-only version of the paper along with the author view of the correspondence tab, but this is yet to be designed.

Reply:
I am wondering if this should be a separate ticket?

Reply:
Yeah I agree that all of that should definitely be a separate ticket :+1:

My only question for completing this ticket is about styling.

Reply:
So where does the code for `pubsweet-pending` live?

Reply:
Oops. Scratch that. @g-sam has just shown me, the namespace is defined in the `package.json` associated with each folder within `client` or `server`?

And, as I understand it, everything inside `client` is "pending" us feeding it back up to pubsweet? (except `elife-theme` which is under `elifesciences`

And also every package inside `server` except `submission` itself?

Would it be useful to document the thinking behind this somewhere?

Reply:
New ticket here https://github.com/elifesciences/elife-xpub/issues/298

However (see comment) I propose we hold off on this functionality for the time being- we're missing too much information and allowing a corresponding author to associate themselves with the submission is enough to satisfy the user story we have.